aquarius.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<!-- background class added to change backgrounds according to screen size -->
+<html lang="en" class="background">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png" />
+  <link rel="stylesheet" href="./assets/style.css" />
+  <link rel="stylesheet" href="./assets/reset.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+
+  <title>Aquarius</title>
+
+</head>
+
+<body>
+  <!-- Mobile Responsive Nav bar -->
+  <nav>
```

Comment:
All of this copy and paste will be unnecessary once we learn handlebars

Unfortunately, there is no great way to avoid all of this copy/paste with pure html

One thing you may have considered was using javascript to generate your page and clearing out the contents

You may have also wanted to look in to modifying the background image for the website through javascript

Reply:
I started the initial layout and since I'm not comfortable with JavaScript this was unfortunately how it ended up. I would love to be at the level that I could code the script file but I'm just not there. That is my goal for the next project.